plugins/jira-dashboard-backend/src/index.ts

```diff
@@ -4,9 +4,6 @@
  * @packageDocumentation
  */
 
-export * from './service/router';
 export { jiraDashboardPlugin as default } from './plugin';
-export { searchJira } from './api';
 export type { SearchOptions } from './api';
-export { jqlQueryBuilder } from './queries';
```

Comment:
We can only remove createRouter and RouterOptions, i.e. what is deprecated. The others should be there.

Reply:
OK. I'll put them back.